.github/workflows/deploy-calibnet.yml

```diff
@@ -0,0 +1,162 @@
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+
+      - name: Terraform Init
+        run: terraform init
+
+      - name: Terraform Validate
+        run: terraform validate -no-color
+
+      - name: Terraform Plan
+        id: plan
+        if: github.event_name == 'pull_request'
+        run: terraform plan -no-color -input=false
+        continue-on-error: true
+
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        if: github.event_name == 'pull_request'
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-regex: "^### Forest Calibnet Infrastruture  Plan"
+
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@v2
+        if: github.event_name == 'pull_request' && !contains(steps.plan.outputs.stdout, 'No changes. Your infrastructure matches the configuration.')
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ### Forest Calibnet Infrastructure  Plan: ${{ steps.plan.outcome }}
+
+            <details><summary>Show Plan</summary>
+
+            ```
+            ${{ steps.plan.outputs.stdout }}
+            ```
+
+            </details>
+          edit-mode: replace
+
+      - name: Delete comment
+        uses: detomarco/delete-comments@v1.0.4
+        if: github.event_name == 'pull_request' && contains(steps.plan.outputs.stdout, 'No changes. Your infrastructure matches the configuration.')
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+
+      - name: Terraform Plan Status
+        if: steps.plan.outcome == 'failure'
+        run: exit 1
+
+      - name: Terraform Apply To Create Forest Infrastructure
+        if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+        run: terraform apply -auto-approve -input=false
+
+      - name: Upload Mainnet Host File As Artifacts
+        if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+        uses: actions/upload-artifact@v2
+        with:
+          name: hosts_calibnet
+          path: ansible/hosts_calibnet
+
+  configure-ansible-forest-infrastructure:
+    name: Configure Ansible Forest Infrastructure
+    runs-on: ubuntu-latest
+    needs: deploy-forest-calibnet-infrastructure
+    defaults:
+      run:
+        shell: bash
+        working-directory: ansible
+    env:
+      ANSIBLE_CONFIG: Ansible.cfg
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+      SPACES_REGION: "fra1"
+      DOMAIN_NAME: "forest-calibnet.chainsafe.io"
+      SPACES_BUCKET_NAME: "forest-calib-loki-logs"
+      SPACES_ENDPOINT: "fra1.digitaloceanspaces.com"
+      LETSENCRYPT_EMAIL: "samuel@chainsafe.io"
```

Comment:
Not sure if it's the way to go. Shouldn't we have a group email for this kind of stuff?